erpnext/crm/doctype/opportunity/opportunity.py

```diff
@@ -137,6 +138,23 @@ def set_applies_to_details(self):
 			if self.meta.has_field(k) and not self.get(k) or k in force_applies_to_fields:
 				self.set(k, v)
 
+	def set_sales_person_details(self):
+		if not self.sales_person:
+			return
+
+		employee_id = frappe.db.get_value("Sales Person", self.sales_person, ['employee'])
+
+		if not employee_id:
+			return
+
+		employee = frappe.db.get_value("Employee", employee_id, ['cell_number', 'personal_email'], as_dict=1)
```

Comment:
Pick either `company_email` or `personal_email` .. Give precedence to `company_email`